src/pages/graphql/schema/orders/mutations/confirm-return.md

```diff
@@ -0,0 +1,135 @@
+---
+title: confirmReturn mutation
+edition: ee
+---
+
+# confirmReturn mutation
+
+The `confirmReturn` mutation confirms a return request from a guest customer. The merchant subsequently decides whether to accept or reject the request.
+
+## Syntax
+
+```graphql
+mutation {
+  confirmReturn(input: ConfirmReturnInput!): RequestReturnOutput
+}
+```
+
+## Reference
```

Comment:
Comment out this section. I'll uncomment it when we autogenerate the reference docs for beta2.